ehrapy/plot/_survival_analysis.py

```diff
@@ -293,5 +296,170 @@ def kaplan_meier(
+        show: Show the plot, do not return figure and axis.
+        title: Set the title of the plot.
+
+    Examples:
+        >>> import ehrapy as ep
+        >>> adata = ep.dt.mimic_2(encoded=False)
+        >>> adata_subset = adata[:, ["mort_day_censored", "censor_flg", "gender_num", "afib_flg", "day_icu_intime_num"]]
+        >>> coxph = ep.tl.cox_ph(adata_subset, event_col="censor_flg", duration_col="mort_day_censored")
+        >>> ep.pl.cox_ph_forestplot(adata_subset)
+
+        .. image:: /_static/docstring_previews/coxph_forestplot.png
+
+    """
+    if uns_key not in adata.uns:
+        raise ValueError(f"Key {uns_key} not found in adata.uns. Please provide a valid key.")
+
+    coxph_fitting_summary = adata.uns[
+        uns_key
+    ]  # pd.Dataframe with columns: coef, exp(coef), se(coef), z, p, lower 0.95, upper 0.95
+    auc_col = "coef"
+
+    if labels is None:
+        labels = coxph_fitting_summary.index
+    tval = []
+    ytick = []
+    for row_index in range(len(coxph_fitting_summary)):
+        if not np.isnan(coxph_fitting_summary[auc_col][row_index]):
+            if (
+                (isinstance(coxph_fitting_summary[auc_col][row_index], float))
+                & (isinstance(coxph_fitting_summary["coef lower 95%"][row_index], float))
+                & (isinstance(coxph_fitting_summary["coef upper 95%"][row_index], float))
+            ):
+                tval.append(
+                    [
+                        round(coxph_fitting_summary[auc_col][row_index], decimal),
+                        (
+                            "("
+                            + str(round(coxph_fitting_summary["coef lower 95%"][row_index], decimal))
+                            + ", "
+                            + str(round(coxph_fitting_summary["coef upper 95%"][row_index], decimal))
+                            + ")"
+                        ),
+                    ]
+                )
+            else:
+                tval.append(
+                    [
+                        coxph_fitting_summary[auc_col][row_index],
+                        (
+                            "("
+                            + str(coxph_fitting_summary["coef lower 95%"][row_index])
+                            + ", "
+                            + str(coxph_fitting_summary["coef upper 95%"][row_index])
+                            + ")"
+                        ),
+                    ]
+                )
+            ytick.append(row_index)
+        else:
+            tval.append([" ", " "])
+            ytick.append(row_index)
+
+    x_axis_upper_bound = round(((pd.to_numeric(coxph_fitting_summary["coef upper 95%"])).max() + 0.1), 2)
+
+    x_axis_lower_bound = round(((pd.to_numeric(coxph_fitting_summary["coef lower 95%"])).min() - 0.1), 1)
+
+    fig = plt.figure(figsize=fig_size)
+    gspec = gridspec.GridSpec(1, 6)
+    plot = plt.subplot(gspec[0, 0:4])  # plot of data
+    tabl = plt.subplot(gspec[0, 4:])  # table
```

Comment:
```suggestion
    table = plt.subplot(gspec[0, 4:])  # table
```

Why not that? Does it shadow some name? And the comments here are then only mildly useful :)

Reply:
refactored the name, thank you for pointing this out